packages/backend/package.json

```diff
@@ -55,7 +56,7 @@
         "@types/sshpk": "^1.17.1",
         "ajv": "^8.3.0",
         "ajv-formats": "^2.1.0",
-        "bcrypt": "^5.0.1",
+        "bcrypt": "^5.1.1",
```

Comment:
bcrypt is depending on minizlib which depends on zlib

Reply:
I'm not 100% sure this will fix the issue, but 5.1.0 did upgrade its depedencies